packages/telemed-ehr/app/src/telemed/features/appointment/AppointmentFooterButton.tsx

```diff
@@ -139,27 +145,19 @@ export const AppointmentFooterButton: FC<AppointmentFooterButtonProps> = (props)
           response={onAssignMe}
           actionButtons={{
             proceed: {
-              text: 'Assign to me',
+              text: 'Assign me',
             },
             back: { text: 'Cancel' },
           }}
         >
           {(showDialog) => (
-            <LoadingButton
+            <FooterButton
               loading={changeTelemedAppointmentStatus.isLoading || isAppointmentLoading}
               onClick={showDialog}
               variant="contained"
-              sx={{
-                textTransform: 'none',
-                fontSize: '15px',
-                fontWeight: 700,
-                borderRadius: 10,
-                backgroundColor: theme.palette.primary.light,
-                '&:hover': { backgroundColor: darken(theme.palette.primary.light, 0.125) },
-              }}
             >
               Assign to me
```

Comment:
The buttons should both say "Assign me" or "Assign to me". Right now, one says "Assign to me" and the other "Assign me"